wormhole-connect/src/views/Bridge/Inputs.tsx/Inputs.tsx

```diff
@@ -181,7 +185,7 @@ function Inputs(props: Props) {
                       selected={selectedNetwork}
                       error={!!(showErrors && props.networkValidation)}
                       onClick={props.onNetworkClick}
-                      editable
+                      editable={!isTransactionInProgress}
```

Comment:
Leave editable as true, and use `disabled={!isTransactionInProgress}`.  Maybe we should change `editable` prop to `filled` or `colorFilled`...